client/lobbies/lobby.tsx

```diff
@@ -154,7 +155,7 @@ function renderChatMessage(msg: SbMessage) {
   }
 }
 
-interface LobbyProps {
+interface LobbyProps extends WithTranslation {
```

Comment:
I would not suggest doing this, as these props are more intended to be the externally visible API of a component (e.g. you could use them to typecheck props you intend to forward to a `Lobby`). You should instead add it in the generic, e.g. `class Lobby extends React.Component<LobbyProps & WithTranslation>`